.sclng/metadata.toml

```
@@ -0,0 +1,8 @@
+version = "1.0.0"
+owner = "etienne@scalingo.com"
+team = "IST"
+description = "Python buildpack"
```

Why not using the same kind of description for all buildpacks?

```suggestion
description = "Official Scalingo Python buildpack"
```